frontend/javascripts/dashboard/explorative_annotations_view.tsx

```diff
@@ -819,8 +815,8 @@ function TopBar({
         width: 200,
         float: "right",
       }}
-      onPressEnter={handleSearchPressEnter}
-      onChange={handleSearch}
+      onSearch={handleOnSearch}
```

Comment:
This component is the only one that does something on "ENTER". I switched to the `<Search>`input `onSearch` props instead since that also handles clicks to the little search icon button, ENTER key etc